webapp/templates/jury/clarification.html.twig

```diff
@@ -61,7 +61,7 @@ Jury
     </span>
 <span class="clarification-subject-form" data-current-selected-subject="{{ clar.categoryid }}" data-clarification-id="{{clar.clarid}}">
 <form action="{{ path('jury_clarification_change_subject', {'clarId': clar.clarid }) }}" method="post" class="form-inline">
-<select name="subject" id="subject" class="custom-select custom-select-sm">
+<select name="subject" class="subject custom-select custom-select-sm">
```

Comment:
Do the classes "subject" and "queue" actually exist?

Reply:
I think the id(subject) was never used in CSS or JS, but was repeated (hence changing of attribute), this just leaves the possibility to use it. I can recheck if this is indeed not used and remove the attribute completely?

Reply:
But your change adds it as a class of the tag?

Reply:
The select tag is used multiple times on the same name (resulting in multiple DOMelements with the same ID) so I kept the identifier by introducing it as a class. I think I checked and the ID was never used anywhere (so could also be removed) or I forgot to check in a change to the JS/CSS where the identifier should be renamed to ".subject" instead of "#subject"